src/helpers/env.ts

```diff
@@ -62,6 +62,57 @@ const prodChecks: [
   ["DENO_DEPLOYMENT_ID", "is truthy"],
 ];
 
+// platformDeployChecks are a series of predicates that attempt to check whether
+// we're deployed outside of localhost testing.  This extends prodChecks with
+// platform specific checks, ensuring that if you deploy to eg. vercel without
+// NODE_ENV=production we still use prod mode.
+const platformDeployChecks: [
+  key: string,
+  customCheck: keyof typeof checkFns,
+  value?: string
+][] = [
+  // Extend prod checks, then check if we're deployed to a platform.
+  [envKeys.IsVercel, "is truthy"],
+  [envKeys.IsNetlify, "is truthy"],
+  [envKeys.IsRender, "is truthy"],
+  [envKeys.RailwayBranch, "is truthy"],
+  [envKeys.IsCloudflarePages, "is truthy"],
+];
+
+const skipDevServerCHecks = prodChecks.concat(platformDeployChecks);
+
+/**
+ * Returns `true` if we're running in production or on a platform, based off of
+ * either passed environment variables or `process.env`.
+ */
+export const skipDevServer = (
+  /**
+   * The optional environment variables to use instead of `process.env`.
+   */
+  env: Record<string, unknown> = allProcessEnv()
+): boolean => {
+  return skipDevServerCHecks.some(([key, checkKey, expected]) => {
+    return checkFns[checkKey](stringifyUnknown(env[key]), expected);
+  });
+};
+
+
+/**
+ * Returns `true` if we believe the current environment is a platform, eg>
+ * Vercel, Render, Railway, Netlify, etc.
+ */
+export const isPlatformDeploy = (
```

Comment:
Can this entire function be removed now? Doesn't look like it's being used anymore.

Reply:
It can be, though given we have the checks for it I figured we should expose them separately?  Dont mind either way!